src/components/SearchResult.js

```diff
@@ -33,6 +33,7 @@ const SearchResult = props => (
                 <td>{row.roomId}</td>
                 <td>{row.checkInDate}</td>
                 <td>{row.checkOutDate}</td>
+                <td >{parseInt((new Date(row.checkOutDate) - new Date(row.checkInDate)) / (1000 * 60 * 60 * 24))} days</td>
```

Comment:
I actually quite like this approach, it's pretty readable. And good job on splitting up the time into readable pieces 👍

However you might want to look into the moment.js library. It has a `.diff` method which is helpful for this sort of thing